src/tbbbind/CMakeLists.txt

```diff
@@ -12,9 +12,6 @@
 # See the License for the specific language governing permissions and
 # limitations under the License.
 
-if (DEFINED CMAKE_SKIP_BUILD_RPATH)
-    set(CMAKE_SKIP_BUILD_RPATH_OLD_VALUE ${CMAKE_SKIP_BUILD_RPATH})
-endif()
```

Comment:
these are removed, because `add_subdirectory` has its own scope and values don't need to be restored